cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/support/cookie/TicketGrantingCookieProperties.java

```diff
@@ -0,0 +1,45 @@
+package org.apereo.cas.configuration.model.support.cookie;
+
+/**
+ * Configuration properties class for tgc.
+ *
+ * @author Dmitriy Kopylenko
+ * @since 5.0.0
+ */
+
+public class TicketGrantingCookieProperties extends AbstractCookieProperties {
+
+    private int rememberMeMaxAge = 1209600;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['1209600' is a magic number.](https://www.codacy.com/app/mmoayyed/cas/file/2768422546/issues/source?bid=3256115&fileBranchId=3357337#l12)